pybandits/base.py

```diff
@@ -74,6 +75,47 @@ class Strategy(PyBanditsBaseModel, ABC):
     Strategy to select actions in multi-armed bandits.
     """
 
+    epsilon: Optional[Float01]
+    default_action: Optional[ActionId]
+
+    def __init__(self, **data: Any) -> None:
+        super().__init__(**data)
+        if self.epsilon is not None:
+            self.select_action = self._make_epsilon_greedy(self.epsilon, self.default_action, self.select_action)
+
+    @staticmethod
+    @validate_arguments
+    def _make_epsilon_greedy(epsilon: Float01, default_action: Optional[ActionId], select_action: Callable):
+        """
+        Wraps a select_action function with epsilon-greedy strategy.
```

Comment:
Can you extend the docstring specifying what is this function doing? 
I mean the logic: `return default_action if default_action else np.random.choice(list(p.keys()))`